styles/styles.css

```
@@ -13,19 +15,178 @@
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --heading-font: 'Merriweather', serif;
+    --body-font: 'Roboto', sans-serif;
+    --color-purple: #5722CD;
+    --color-blue: #4845E4;
+    --color-pink: #ED4385;
+    --color-text: #0C0C0C; /*zwart - standaard tekstkleur*/
+    --color-background-words: #F2F0FF; /*lichtpaars - voor "hello" en "services in de achtergrond")*/
+    --color-background: #F3F6F9; /*lichtgrijs - voor achtergrond van de projecten*/
+    --color-service-tile: #7A7A7A; /*grijs - voor service-tegel tekst*/
+}
+
 
 /* ------------------------------- 2. Typography ----------------------------------- */
+h1, h2, h3, h4, h5, h6 {
+    font-family: var(--heading-font);
+    /*color: var(--color-text);*/
+}
+
+body {
+    font-family: var(--body-font);
+    color: var(--color-text);
+}
 
 /* ------------------------------- 3. Layout --------------------------------------- */
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 20px;
+
+}
+
+
+
+
 
 /* --------------- 3.1 Reusables ----------------------- */
 
+.work {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-basis: 200px;
+}
+
+.portfolio-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    text-align: left;
+    padding: 50px;
+    width: 400px;
+    flex:3;
+}
+
+.portfolio-image {
+    display: flex;
+    align-items: center;
+    border: red dotted 2px;
+    padding: 100px;
+    /*box-shadow: 0px 4px 8px grey;*/
+    flex: 5;
+}
+
+
```

```suggestion

.portfolio-image img {
    width: 100%;
}
```

Dan schaalt de afbeelding netjes bij smaller maken van het scherm.